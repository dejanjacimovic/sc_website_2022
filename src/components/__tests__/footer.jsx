import React from 'react';
import renderer from 'react-test-renderer';
import { Link } from 'gatsby';
import Footer from '../footer';

describe('Footer', () => {
  it('renders correctly', () => {
    const submenuTitle = renderer
      .create(<Footer h4="Company" />)
      .toJSON();

    const pricingLink = renderer
      .create(<Link to="/pricing/" />)
      .toJSON();

    expect(submenuTitle).toMatchSnapshot();
    expect(pricingLink).toMatchSnapshot();
  });
});
