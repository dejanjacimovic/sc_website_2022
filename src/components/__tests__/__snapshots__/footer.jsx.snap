// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="bg-gray-900 font-sans text-gray-900 antialiased"
>
  <div
    className="mx-auto max-w-screen-xl py-12 px-4 sm:px-6 lg:py-16 lg:px-8"
  >
    <div
      className="grid grid-cols-2 gap-8 md:grid-cols-6"
    >
      <div
        className="sm:mt-12"
      >
        <h4
          className="text-sm font-semibold uppercase leading-5 tracking-wider text-gray-400"
        >
          Company
        </h4>
        <ul
          className="mt-4"
        >
          <li>
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="/testimonials/"
            >
              Testimonials
            </a>
          </li>
          <li
            className="mt-4"
          >
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="/about/"
            >
              About us
            </a>
          </li>
          <li
            className="mt-4"
          >
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="/services-ecommerce-development/"
            >
              Services
            </a>
          </li>
          <li
            className="mt-4"
          >
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="/pricing/"
            >
              Pricing
            </a>
          </li>
          <li
            className="mt-4"
          >
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="https://stuntcoders.careers/"
              target="_blank"
            >
              Careers
            </a>
          </li>
        </ul>
      </div>
      <div
        className="sm:mt-12"
      >
        <h4
          className="text-sm font-semibold uppercase leading-5 tracking-wider text-gray-400"
        >
          Legal
        </h4>
        <ul
          className="mt-4"
        >
          <li>
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="/privacy-policy/"
            >
              Privacy
            </a>
          </li>
          <li
            className="mt-4"
          >
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="/terms/"
            >
              Terms
            </a>
          </li>
          <li
            className="mt-4"
          >
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="/refund-policy/"
            >
              Refund
            </a>
          </li>
          <li
            className="mt-4"
          >
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="/cancellation-policy/"
            >
              Cancellation
            </a>
          </li>
          <li
            className="mt-4"
          >
            <a
              className="text-base leading-6 text-gray-300 hover:text-white"
              href="/restrictions-policy/"
            >
              Restrictions
            </a>
          </li>
        </ul>
      </div>
      <div
        className="sm:mt-12"
      >
        <h5
          className="text-sm font-semibold uppercase leading-5 tracking-wider text-gray-400"
        >
          United States
        </h5>
        <p
          className="mt-4 text-base leading-6 text-gray-300"
        >
          8 The Green
          <br />
          19901 Dover
          <br />
          Delaware
          <br />
          <br />
          +1 (424) 297-1577 (Dejan)
          <br />
          <br />
        </p>
      </div>
      <div
        className="sm:mt-12"
      >
        <h5
          className="text-sm font-semibold uppercase leading-5 tracking-wider text-gray-400"
        >
          Serbia
        </h5>
        <p
          className="mt-4 text-base leading-6 text-gray-300"
        >
          Vatroslava Lisinskog 21
          <br />
          11000 Beograd
          <br />
        </p>
      </div>
      <div
        className="sm:mt-12"
      >
        <h5
          className="text-sm font-semibold uppercase leading-5 tracking-wider text-gray-400"
        >
          Nordics
        </h5>
        <p
          className="mt-4 text-base leading-6 text-gray-300"
        >
          Kalevankatu 31
          <br />
          00100, Helsinki
          <br />
          Finland
          <br />
          <br />
          +358 413131372 (Hugo)
          <br />
          <br />
        </p>
      </div>
      <img
        className="w-32 object-right-top sm:m-12"
        src="https://stcdshare.s3.amazonaws.com/aaa-banner.png"
      />
    </div>
    <div
      className="mt-8 border-t border-gray-700 pt-8 md:flex md:items-center md:justify-between"
    >
      <p
        className="mt-8 text-base leading-6 text-gray-400 md:order-1 md:mt-0"
      >
        Â© 2020 Adorable Corporation, Inc. in collaboration with StuntCoders doo. All rights reserved.
      </p>
    </div>
  </div>
</div>
`;

exports[`Footer renders correctly 2`] = `
<a
  href="/pricing/"
/>
`;
